### PR TITLE
Soni

### PR DESCRIPTION
**Pull request by liangyingz**

### CODE REVIEW

File: flow/flow.qaqc.plau/flow.qaqc.plau.R [hunk 1]
```diff
@@ -130,6 +130,8 @@
 library(foreach)
 library(doParallel)
 
+source("/Users/lzhang/Downloads/def.plau.R")
```
Comment (covesturtevant) [inline flow/flow.qaqc.plau/flow.qaqc.plau.R:133]: Please remove

**Review by covesturtevant — COMMENTED** [review]
Just one line to remove - see comment. Once you do, feel free to merge yourself. Thanks!

File: flow/flow.qaqc.plau/flow.qaqc.plau.R (hunk 1 above)
Comment (liangyingz) [reply flow/flow.qaqc.plau/flow.qaqc.plau.R:133]: Ooops. Thanks for catching that!

**Review by rmarkel-neon — APPROVED** [review]
It looks fine to me.

**Pull request merged**